components/input-number/style/index.ts

```diff
@@ -263,10 +263,6 @@ const genInputNumberStyles: GenerateStyle<InputNumberToken> = (token: InputNumbe
           [`${componentCls}-handler-wrap`]: {
             display: 'none',
           },
-
-          [`${componentCls}-input`]: {
-            color: 'inherit',
-          },
```

Comment:
need revert ~

Reply:
I've overridden this file with the one from the master branch.